docs/cookbook/creating_landing_page_blocks_(enterprise).md

```diff
@@ -263,163 +240,60 @@ YUI.add('ezs-addcustomblockplugin', function (Y) {
-use EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Definition\BlockAttributeDefinition;
-use EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Model\AbstractBlockType;
-use EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Model\BlockType;
-use EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Model\BlockValue;
-
-/**
- * Tag block
- * Renders simple HTML.
- */
-class TagBlock extends AbstractBlockType implements BlockType
-{
-    /**
-     * Returns array of parameters required to render block template.
-     *
-     * @param array $blockValue Block value attributes
-     *
-     * @return array Template parameters
-     */
-    public function getTemplateParameters(BlockValue $blockValue)
-    {
-        return ['block' => $blockValue];
-    }
-
-    /**
-     * Creates BlockDefinition object for block type.
-     *
-     * @return \EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Definition\BlockDefinition
-     */
-    public function createBlockDefinition()
-    {
-        return new BlockDefinition(
-            'tag',
-            'Tag Block',
-            'default',
-            'bundles/ezsystemslandingpagefieldtype/images/thumbnails/tag.svg',
-            [],
-            [
-                new BlockAttributeDefinition(
-                    'content',
-                    'Content',
-                    'text',
-                    '/[^\\s]/',
-                    'Provide html code'
-                ),
-            ]
-        );
-    }
-
-    /**
-     * Checks if block's attributes are valid.
-     *
-     * @param array $attributes
-     *
-     * @throws \EzSystems\LandingPageFieldTypeBundle\Exception\InvalidBlockAttributeException
-     */
-    public function checkAttributesStructure(array $attributes)
-    {
-        if (!isset($attributes['content'])) {
-            throw new InvalidBlockAttributeException('Tag', 'content', 'Content must be set.');
-        }
-    }
-}
-```
-
-!!! tip
-
-    If you want to make sure that your block is only available in the Element menu in a specific situation, you can override the `isAvailable` method, which makes the block accessible by default:
-
-    ``` php
-    public function isAvailable()
-        {
-            return true;
-        }
-    ```
-
-#### service.yml configuration
-
-``` yaml
-# services.yml
-ezpublish.landing_page.block.tag:
-    class: EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Model\Block\TagBlock
-    tags:
-        - { name: landing_page_field_type.block_type, alias: tag }
-```
-
-#### Block template
-
-`{{ block.attributes.content|raw }}`
-
 ## Defining Landing Page blocks in the configuration file
 
-There is an alternative way to define/configure Landing Page block types that requires modifying the `config.yml` in an application or a bundle and using the `ez_systems_landing_page_field_type` key.
+There is a faster and more developer-friendly way to create Landing Page block types
```

Comment:
What is the benefit for the developer? (Can we demonstrate it's "more developer-friendly?)